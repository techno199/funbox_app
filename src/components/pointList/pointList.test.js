import React from 'react'
import { PointList } from '.'
import renderer from 'react-test-renderer'

test('should render point list', () => {
  let items = [
    { id: 1, value: 'a' },
    { id: 2, value: 'b' },
    { id: 3, value: 'c' }
  ]

  const handleRemoveClick = jest.fn(id => () => {})
  const component = renderer.create(
    <PointList items={items} onRemoveClick={handleRemoveClick}/>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
