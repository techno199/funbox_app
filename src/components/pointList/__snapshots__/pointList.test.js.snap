// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render point list 1`] = `
<ul
  className="MuiList-root-4 MuiList-padding-5 PointListUnstyled-root-1"
/>
`;
